components/sections/Footer/index.tsx

```diff
@@ -0,0 +1,107 @@
+import Image from 'next/image';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import NavItem from '@/components/sections/NavItem';
+
+import styles from './index.module.css';
+
+const Footer: FC = () => {
+  const externalLinks = [
+    {
+      href: 'https://openjsf.org/wp-content/uploads/sites/84/2021/01/OpenJS-Foundation-Trademark-Policy-2021-01-12.docx.pdf',
+      translationId: 'trademarkPolicy',
```

Comment:
it's should be something like that `components.sections.footer.link.trademarkPolicy` or `components.sections.footer.extLink.trademarkPolicy`